modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.js

```diff
@@ -46,11 +48,17 @@ function CreationMenu({primaryItems}) {
 										key={i}
 										onClick={(event) => {
 											event.preventDefault();
+
 											setActive(false);
-											triggerAction(
-												item,
-												frontendDataSetContext
-											);
+
+											item.onClick?.();
```

Comment:
I didn't know `?.()` was an allowed syntax. I thought it would be `?()`

Reply:
Yep, `?.` is technically the operator. With that in mind, it's kinda weird that object property chaining is not `obj?..foo` :slightly_smiling_face: